pkg/process/checks/process.go

```diff
@@ -147,6 +150,9 @@ func (p *ProcessCheck) Init(syscfg *SysProbeConfig, info *HostInfo, oneShot bool
 
 	p.initConnRates()
 
+	p.serviceExtractor = parser.NewServiceExtractor(true, true)
```

Comment:
I think its a bit cleaner here to use variables instead of literal booleans. Makes it more clear what you trying to do.

Reply:
updated!